readthedocs/subscriptions/templates/subscriptions/notifications/organization_disabled_email.txt

```diff
@@ -0,0 +1,14 @@
+{% extends "core/email/common.txt" %}
+Your subscription period for "{{ organization.name }}" has ended and you haven't yet renewed.
+If you'd like to continue using Read the Docs to manage your
+documentation, please renew the subscription for your organization now.
```

Comment:
I'm assuming these .txt templates are just a copy and paste of the .html ones but without the HTML tags, is this correct?